### PR TITLE
LPS-189950 | master

### PR DESCRIPTION
**Pull request by melodywu**
https://liferay.atlassian.net/browse/LPS-189950
@JoyceWang08211 thanks for helping review the NAT

### CODE REVIEW

**Comment by melodywu** [comment]
ci:test:sf

**Comment by melodywu** [comment]
ci:test:staging

File: portal-web/test/functional/com/liferay/portalweb/macros/site/SiteTemplates.macro [hunk 1]
```diff
@@ -474,7 +474,23 @@ definition {
...
+		if (isSet(site)) {
+			JSONRole.addSiteRole(
+				roleKey = "Access Site",
+				roleTitle = "Access Site");
+		}
+		else {
+			JSONRole.addRegularRole(roleTitle = "Access Site");
+		}
+
+		if (isSet(site)) {
```
Comment (JoyceWang08211) [inline portal-web/test/functional/com/liferay/portalweb/macros/site/SiteTemplates.macro:486]: Line 486-Line 493: It's duplicated with the previous condition.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/sitetemplates/cpsitetemplates/CPSitetemplates.testcase [hunk 2]
```diff
@@ -1652,18 +1652,19 @@ definition {
...
+				value1 = "Warning:The friendly URL of this page is conflicting with a friendly URL of a page in the site template, from which this site was created: Page Page1 of Site Template Name (Please contact the administrator to resolve this friendly URL conflict.)");
 		}
 	}
 
 	@description = "This is a use case for LPS-174434 TC-8. Verify there's a confirmation dialog when a friendly URL collision exists and Administrator tries to edit an existing Site page."
-	@ignore = "true"
 	@priority = 4
 	test ViewConfirmationDialogOfEditExistingSitePageWithURLCollision {
-		property custom.properties = "feature.flag.LPS-174434=true";
+		property custom.properties = "feature.flag.LPS-174417=true";
```
Comment (JoyceWang08211) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/sitetemplates/cpsitetemplates/CPSitetemplates.testcase:1667]: The current test is covered by CPSitetemplates#ViewWarnignMessageOfEditExistingSitePageWithURLCollision. Please remove the current test.

**Review by JoyceWang08211 — CHANGES_REQUESTED** [review]

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/sitetemplates/cpsitetemplates/CPSitetemplates.testcase (hunk 2 above)
Comment (melodywu) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/sitetemplates/cpsitetemplates/CPSitetemplates.testcase:1667]: ok

**Comment by melodywu** [comment]
ci:test:sf

**Comment by melodywu** [comment]
ci:test:staging

**Comment by JoyceWang08211** [comment]
LGTM

**Comment by JoyceWang08211** [comment]
ci:forward

**Pull request closed without merging**